lib/generate.js

```diff
@@ -68,6 +68,10 @@ function transformValue(value, field, spec) {
 				value = value.replace(nonAsciiRegex, '*');
 			if (!spec.supportsNewLineChars)
 				value = value.replace(newLineRegex, ' ');
+			if (!spec.supportsExtraRecordEndingChars) {
+				var recordEndingRegex = new RegExp(`[${spec.recordEnding}]`, 'g');
+				value = value.replace(recordEndingRegex, '*');
+			}
```

Comment:
Make sure this can all go on one line so no braces are needed and the existing style is maintained - shift out the logic into another function if necessary